modules/configuration/src/testFixtures/java/org/apache/ignite/internal/configuration/TestConfigurationChanger.java

```diff
@@ -51,6 +53,13 @@ public TestConfigurationChanger(
         this.generator = generator;
     }
 
+    @Override
+    public void start() {
+        super.start();
+
+        assertThat(onDefaultsPersisted(), willCompleteSuccessfully());
```

Comment:
(Minor) Let's remove redundant assertions from tests? I see just 7 of them

Reply:
Done